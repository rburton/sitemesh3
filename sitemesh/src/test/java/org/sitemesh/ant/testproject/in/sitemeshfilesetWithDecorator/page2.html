<html>
    <body>I am a simple page 2</body>
</html>