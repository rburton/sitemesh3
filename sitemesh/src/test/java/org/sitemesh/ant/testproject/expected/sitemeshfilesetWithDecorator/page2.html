<html>
    <body>DECORATOR{{{I am a simple page 2}}}</body>
</html>